src/client/_clientwindow.py

```diff
@@ -870,12 +874,26 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def show_autojoin_settings_dialog(self):
+        autojoin_channels_list = []
+        autojoin_channels_list = Settings.get('chat/auto_join_channels')
+        text_of_autojoin_settings_dialog = """
+        Autojoin channels list : """ + ';'.join(autojoin_channels_list) + """
+        Enter the list of channels you want to join, separated by ;
+        For example: #poker;#newbie
+        To disable autojoin channels feature, leave the box empty and press ok.
+        """
+        channels_input_of_user, ok = QInputDialog.getText(None, 'Set autojoin channels',text_of_autojoin_settings_dialog, QLineEdit.Normal, "")
```

Comment:
If you pass the current widget (self) instead of None as the first argument, the dialog will be themed consistently with the rest of the client.

Reply:
done

Reply:
done, didn't know that it was for that. Was always a mystery this thing.